### PR TITLE
GSG: initial documents to support CaaS, CIC announcement

### PR DESCRIPTION
**Pull request by ttzeng**
This commit adds initial documents to support the CaaS (Celadon as a Service) and CIC (Celadon in Container) software launch.

Signed-off-by: Wang, Jian W <jian.w.wang@intel.com>
Signed-off-by: Liu, Xinwu <xinwu.liu@intel.com>
Signed-off-by: Zheng, Joe <joe.zheng@intel.com>
Signed-off-by: Tonny Tzeng <tonny.tzeng@intel.com>

### CODE REVIEW

**Comment by jian150** [comment]
fine by me

File: source/getting_started/on-container.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+Introduction
+------------
+
+The goal of **Celadon in Container (CIC)** feature is to run the |C| Android image in a Docker container, so that you can run the image on Linux devices through Docker tools, and run Android applications in it.
+
+CIC should be able to run on modern PCs with Intel® processors which have Intel 6th Generation or later GPU integrated. The |NUC| model `NUC7i7BNH <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc7i7bnh.html>`_ and model `NUC7i5BNH <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc7i5bnh.html>`_ are recommended to try out the CIC features.
+
+.. note::
+   The CIC is currently of Version 0.5, whcih provides a preview of the feature for pilot and development purposes. Some features such as Trusty, Verified Boot, and OTA update are not included in this preview version. Those features is planned for the upcoming releases.
```
Comment (joez) [inline source/getting_started/on-container.rst]: - typo here: "whcih"
- "Intel® processors which have Intel 6th Generation or later GPU integrated" should be "Intel® 6th generation or later processors with integrated GPU"
- "Those features is planned" should be "Those features are planned"

File: source/getting_started/on-container.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Set up Docker Engine
+--------------------
+
+You need to install Docker on both the development host and the target device. Enter the following commands to install the prerequisites, set up the repository, and install the Docker Engine. Reference `Get Docker Engine - Community for Ubuntu <https://docs.docker.com/install/linux/docker-ce/ubuntu/>`_ installation guide for more detailed information.
```
Comment (joez) [inline source/getting_started/on-container.rst]: better to change "Reference" to "Refer to"

File: source/getting_started/on-container.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+.. code-block:: bash
+
+    $ sudo apt-get install apt-transport-https ca-certificates curl
+    $ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    $ sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+    $ sudo apt-get update
+    $ sudo apt-get install -y docker-ce docker-ce-cli containerd.io
+    $ sudo usermod -aG docker $USER
+
+The last command is optional if you want to run the Docker as a non-root user.
```
Comment (joez) [inline source/getting_started/on-container.rst]: this statement is a little confusing, shall we change to like this:

```
If you would like to use Docker as a non-root user:
$ sudo usermod -aG docker $USER
Remember to log out and back in for this to take effect!
```

File: source/getting_started/on-container.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+On the target device, CIC currently requires Linux kernel version 4.14.20 or later, which is available in most Linux distributions such as Clear Linux, Rancher OS, and Ubuntu Linux, etc. The previous setup instructions are based on Ubuntu 16.04 LTS distribution.
```
Comment (joez) [inline source/getting_started/on-container.rst]: change "previous setup instructions" to "current setup instructions"?

File: source/getting_started/on-container.rst [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+Reference the :ref:`build-from-source` section in the Getting Started Guide to set up the |C| source tree and the build environment. There are two build targets associated with the CIC builds:
+
+    **cic**
+        target to compliant with Android CDD
+
+    **cic_dev**
+        target for development purposes (available on the CIC branch of the |C| Android-P release)
+
+The following commands select **cic-userdebug** as the lunch target and start the build. The CIC package will be generated at **out/target/product/celadon/cic-${USER}.tar.gz**.
```
Comment (joez) [inline source/getting_started/on-container.rst]: "out/target/product/celadon/cic-${USER}.tar.gz" is not precise, better to use "$OUT/$TARGET_PRODUCT-*.tar.gz"

File: source/getting_started/on-container.rst [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+.. code-block:: bash
+
+    $ source build/envsetup.sh
+    $ lunch cic-userdebug
+    $ make cic -j $(nproc)
```
Comment (joez) [inline source/getting_started/on-container.rst]: "$(nproc)" is not necessary, "make -j cic" will have the same result

File: source/getting_started/build-source.rst [hunk 7]
```diff
@@ -67,23 +90,42 @@ Optionally, delete existing output of any previous build with the following comm
...
-* Build the kernelflinger executables
+.. code-block:: bash
 
-    .. code-block:: bash
+    $ make SPARSE_IMG=true flashfiles -j $(nproc)
 
-        $ make SPARSE_IMG=true flashfiles -j $(nproc)
+.. note::
+    In case of the CIC build, the generated image is packaged at the following location:
+    **out/target/product/celadon/cic-${USER}.tar.gz**
```
Comment (joez) [inline source/getting_started/build-source.rst]: "out/target/product/celadon/cic-${USER}.tar.gz" is not precise, better to use "$OUT/$TARGET_PRODUCT-*.tar.gz"

File: source/getting_started/on-container.rst (hunk 1 above)
Comment (ttzeng) [reply source/getting_started/on-container.rst]: @joez, fixed and thanks.

**Pull request merged**